src/Microsoft.AspNetCore.Rewrite/Internal/UrlRewrite/InputParser.cs

```diff
@@ -3,11 +3,9 @@
 
 using System;
 using System.Collections.Generic;
-using System.Text.Encodings.Web;
-using Microsoft.AspNetCore.Rewrite.Internal;
-using Microsoft.AspNetCore.Rewrite.UrlRewrite.PatternSegments;
+using Microsoft.AspNetCore.Rewrite.Internal.UrlRewrite.PatternSegments;
 
-namespace Microsoft.AspNetCore.Rewrite.UrlRewrite
+namespace Microsoft.AspNetCore.Rewrite.Internal.UrlRewrite
```

Comment:
In the future mechanical changes like updating namespaces should be done in their own PR. Reviewing 110 files is intimidating.